.github/workflows/templates-test.yml

```diff
@@ -121,18 +127,18 @@ jobs:
       - run: tar -xf create-tauri-app/create-tauri-app.tar
 
       - name: bootstrap a ${{ matrix.settings.template }} project (${{ matrix.settings.flags }})
-        run: ./target/release/cargo-create-tauri-app tauri-app -m ${{ matrix.settings.manager }} -t ${{ matrix.settings.template }} ${{ matrix.settings.flags }} -y
+        run: ./target/release/cargo-create-tauri-app ${{ (matrix.settings.manager == 'dotnet') && 'TauriApp' || 'tauri-app' }} -m ${{ matrix.settings.manager }} -t ${{ matrix.settings.template }} ${{ matrix.settings.flags }} -y
 
       - run: sed -i 's/com.tauri.dev/com.create-tauri-app.dev/g' **/tauri.conf.json
-        working-directory: tauri-app
+        working-directory: ${{ (matrix.settings.manager == 'dotnet') && 'TauriApp' || 'tauri-app' }}
 
       - run: ${{ matrix.settings.install_cmd }}
         if: matrix.settings.install_cmd  != ''
-        working-directory: tauri-app
+        working-directory: ${{ (matrix.settings.manager == 'dotnet') && 'TauriApp' || 'tauri-app' }}
 
       - run: ${{ matrix.settings.run_cmd }} tauri build ${{ matrix.settings.no_bundle_flag }}
         if: matrix.settings.manager != 'npm'
-        working-directory: tauri-app
+        working-directory: ${{ (matrix.settings.manager == 'dotnet') && 'TauriApp' || 'tauri-app' }}
```

Comment:
when bootstrapping a dotnet project it should support `tauri-app` too, we shouldn't handle it here at all, we just need to convert `tauri-app` to `TauriApp` automatically in in the template in the files that require it to be without `-`, I guess for example that would be `Program.cs.lte`, and in that file we change `{% project_name %}` to `{% project_name_upper_case %}`. We will need to inject that variable into the template data though.